.env.test

```diff
@@ -0,0 +1,8 @@
+# Flask server configuration.
+FLASK_APP=app
+FLASK_SKIP_DOTENV=True
+
+# Change the following values for local development.
+APP_API_KEY=eiruty89457934iy
+APP_TOKEN=ijehgeriu835hghe8r9348hfi539h585terh5893
```

Comment:
I can see these aren't real credentials, but it can be nice to make that incredibly obvious so that future developers don't worry about it - e.g. `APP_API_KEY=fake-key` or something